au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksTitleBar.qml

```diff
@@ -0,0 +1,95 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import Muse.Ui
+import Muse.UiComponents
+
+import "qrc:/kddockwidgets/private/quick/qml/" as KDDW
+
+KDDW.TitleBarBase {
+    id: titleBar
+    anchors.fill: parent
+
+    implicitHeight: gripButton.implicitHeight
+    heightWhenVisible: 32
+
+    property var navigationPanel
+    property var navigationOrder
+    property var contextMenuModel
+
+    RowLayout {
+        anchors.fill: parent
+        spacing: 0
+
+        FlatButton {
+            id: gripButton
+
+            Layout.fillHeight: true
+            Layout.preferredWidth: 28
+            width: 28
+            height: 32
+            backgroundRadius: 0
+
+            visible: true
+            normalColor: ui.theme.backgroundSecondaryColor
+            hoverHitColor: ui.theme.buttonColor
+
+            mouseArea.objectName: root.objectName + "_gripButton"
+            mouseArea.cursorShape: Qt.SizeAllCursor
+            // do not accept buttons as FlatButton's mouseArea will override
+            // DockTitleBar mouseArea and panel will not be draggable
+            mouseArea.acceptedButtons: Qt.NoButton
+
+            transparent: false
+            contentItem: StyledIconLabel {
+                iconCode: IconCode.TOOLBAR_GRIP
+            }
+        }
+
+        SeparatorLine {
+            id: spacer
+
+            height: 32
```

Comment:
oh, you don't need these settings, they are already in the component
try using empty `SeparatorLine { }`

Reply:
fixed